store/tikv/backoff.go

```diff
@@ -241,6 +241,9 @@ func (b *Backoffer) WithVars(vars *kv.Variables) *Backoffer {
 	if vars != nil {
 		b.vars = vars
 	}
+	if math.MaxInt32/b.vars.BackOffWeight >= b.maxSleep {
+		b.maxSleep *= b.vars.BackOffWeight
```

Comment:
So all backoff will use this weight? How about specifying only `prewrite`,`commit`,`get_tso` which  will only be used in 2pc?

Reply:
What are the benefits of doing this?

Reply:
I mean, if there is a physical failure, the need to change the backoff should be similar. Still keeping other values fixed, which may make this parameter very useless.